invenio_files_rest/views.py

```diff
@@ -814,6 +839,49 @@ def delete(self, bucket=None, key=None, version_id=None, upload_id=None,
             obj = self.get_object(bucket, key, version_id)
             return self.delete_object(bucket, obj, version_id)
 
+
+class ObjectVersionTagResource(ContentNegotiatedMethodView):
+    """Tag resource."""
+
+    def __init__(self, *args, **kwargs):
+        """Instantiate content negotiated view."""
+        super(ObjectVersionTagResource, self).__init__(*args, **kwargs)
+
+    @pass_bucket
+    @need_bucket_permission('bucket-update')
```

Comment:
Special actions like ``files-rest-objectversion-tag-update`` have been added in ``permissions.py`` but are not used here

Reply:
Looks like a wrong copy paste to me, I will fix permissions.